packages/google_maps_flutter/google_maps_flutter/example/lib/place_marker.dart

```diff
@@ -53,10 +53,11 @@ class PlaceMarkerBodyState extends State<PlaceMarkerBody> {
     final Marker? tappedMarker = markers[markerId];
     if (tappedMarker != null) {
       setState(() {
-        if (markers.containsKey(markerId)) {
-          final Marker resetOld = markers[markerId]!
+        final MarkerId? previousMarkerId = selectedMarker;
+        if (previousMarkerId != null && markers.containsKey(previousMarkerId)) {
+          final Marker resetOld = markers[previousMarkerId]!
               .copyWith(iconParam: BitmapDescriptor.defaultMarker);
-          markers[markerId] = resetOld;
+          markers[previousMarkerId!] = resetOld;
```

Comment:
The '!' will have no effect because the receiver can't be null at example/lib/place_marker.dart:60:35

Reply:
yup just saw that, thanks